app/controllers/groups_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class GroupsController < CollectionsController
```

Comment:
Is this supposed to be a CollectionsController ??  Is this to leave he database migration alone?

Reply:
Yes, I apologize, I can consolidate these.